modelutils.py

```diff
@@ -91,3 +104,57 @@ def get_sequential_groups(model):
         ]
     else:
         raise ValueError(MODEL_ERROR_MSG.format(model.config.model_type))
+
+
+def read_quant_weight_from_file(load_path, block_i, layer_name):
+    return torch.load(load_path + '/' + str(block_i) + '/' + layer_name)
+
+
+def load_quantized_model(model, load_path):
+    layers = get_layers(model)
+    for i in range(len(layers)):
+        layer = layers[i]
+        sub_layers = find_sublayers(layer)
+        for name in sub_layers:
+            dic = read_quant_weight_from_file(load_path, i, name)
```

Comment:
let's rename dic -> state_dict

Reply:
Both names are not that great, I would try to think something else.

Reply:
`quantized_params_dict` fine